.github/workflows/protected_branches.yml

```diff
@@ -21,8 +21,23 @@ jobs:
           miniforge-variant: Mambaforge
           environment-file: conda_development.yml
           use-mamba: true
+      - name: Add status to commit
+        uses: peter-evans/commit-comment@v2
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        with:
+          body: |
+            Github job ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}
```

Comment:
That didn't work https://github.com/ornlneutronimaging/iMars3D/commit/9aa12245d9db98689012e9492b9029d1b3ab41e7#commitcomment-88000389